Utilities/build-script-helper.py

```diff
@@ -131,7 +137,10 @@ def handle_invocation(swift_exec, args):
     bin_path = swiftpm_bin_path(swift_exec, swiftpm_args, env)
     swiftpm_args += ['-Xswiftc', '-no-toolchain-stdlib-rpath']
     swiftpm('build', swift_exec, swiftpm_args, env)
-    install(bin_path, args.toolchain)
+
+    if not args.install_prefixes:
+      args.install_prefixes = [args.toolchain]
```

Comment:
I had tested this default case like this initially, then modified it to just passing `args.install_prefixes if not None else [args.toolchain]` to the line below and not tested the default case again. For whatever reason, that doesn't work, so going back to this.